pkg/server/controller_test.go

```diff
@@ -19,14 +21,52 @@ import (
 )
 
 const assetAtCompressionThreshold, assetLtCompressionThreshold = "AssetAtCompressionThreshold", "AssetLTCompressionThreshold"
+const tlsCert = 
```

Comment:
we could put these strings in files next to the actual code, I think it's a common pattern in go ecosystem, for example we use it here: https://github.com/pyroscope-io/pyroscope/tree/33db1738ccd995ceb5fcbfd6faf16d2b7f78836a/pkg/convert/testdata

Two benefits to this approach:
* no need to create temp files
* allows you to manually test TLS code via command line, e.g `pyroscope server -tls-certificate-file pkg/server/testdata/cert.pem -tls-key-file pkg/server/testdata/key.pem`